utils/preprocessing.py

```diff
@@ -61,7 +61,27 @@ def preprocess_ocean_warming(df: pd.DataFrame) -> pd.DataFrame:
     df.rename(columns={'index': 'date', 'NOAA': 'ocean_warming'}, inplace=True)
     df['date'] = pd.to_datetime(df['date'])
     return df
+def preprocess_renewable_energy_share(df: pd.DataFrame) -> pd.DataFrame:
```

Comment:
Please follow PEP8, and keep two empty lines between functions.

Reply:
Done